core/models/volumes.go

```diff
@@ -55,6 +58,13 @@ type ContainerVolumeSource struct {
 	Image ContainerImage `json:"image"`
 }
 
+// VirtioFSSource represents the details of the VirtioFS volume.
+type VirtioFSVolumeSource struct {
+	// Image is the OCI image to use.
+	Path string `json:"path"`
```

Comment:
I was chatting with someone the other day about flintlock and they really wanted the equivalant of "host path" storage, so this is great.